framework/audio/engine/midisource.cpp

```diff
@@ -48,7 +48,8 @@ struct MidiSource::SLInstance : public SoLoud::AudioSourceInstance {
 
     unsigned int getAudio(float* aBuffer, unsigned int aSamplesToRead, unsigned int /*aBufferSize*/) override
```

Comment:
why is the "bufferSize" unused? Isn't it significant to setup its size?

Reply:
Because we don't need it :) We need a number of samples to read.

Reply:
I mean, if it makes no sense to use it, why it is still here?) Can we remove this argument?

Reply:
Unfortunately, we not possible remove it, because this is a virtual method from SoLoud class.